obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -1478,18 +1495,69 @@ double OBS_API::getCurrentBandwidth(void)
 			kbitsPerSec = double(bitsBetween) / timePassed / 1000.0;
 		}
 
-		lastBytesSent     = bytesSent;
+		lastBytesSent = bytesSent;
 		lastBytesSentTime = bytesSentTime;
+		dataOutput = bytesSent / (1024.0 * 1024.0);
 	}
-
-	return kbitsPerSec;
 }
 
 double OBS_API::getCurrentFrameRate(void)
 {
 	return obs_get_active_fps();
 }
 
+double OBS_API::getAverageTimeToRenderFrame()
+{
+	return (double)obs_get_average_frame_time_ns() / 1000000.0;
+}
+
+std::string OBS_API::getDiskSpaceAvailable()
+{
+#define MBYTE (1024ULL * 1024ULL)
+#define GBYTE (1024ULL * 1024ULL * 1024ULL)
+#define TBYTE (1024ULL * 1024ULL * 1024ULL * 1024ULL)
```

Comment:
We shouldn't re define these every time a call to `getDiskSpaceAvailable()` is made.

Reply:
Will move those defines outside of there

Reply:
Moved those defines outside of the function